src/pages/ReimbursementAccount/EnableStep.js

```diff
@@ -42,6 +47,18 @@ const defaultProps = {
 
 class EnableStep extends React.Component {
     componentDidMount() {
+        this.fetchData();
+    }
+
+    componentDidUpdate(prevProps) {
+        if (prevProps.network.isOffline === this.props.network.isOffline) {
```

Comment:
Sorry I made a mistake in the issues description, but this would actually mean that we also trigger a data fetch when we go offline and we only want it when we are going online. The correct logic should be:

```js
if (prevProps.network.isOffline && !this.props.network.isOffline) {
    this.fetchData();
}
```

Reply:
@marcaaron the linter doesn't like that because it wants you to return early. I think we want to do it like this:

```
componentDidUpdate(prevProps) {
        if (!prevProps.network.isOffline || this.props.network.isOffline) {
            return;
        }

        this.fetchData();
    }
```

Reply:
Nice. Yeah that lint rule is really annoying inside these `componentDidUpdate()` because it's harder to parse. I'll update the main issue description. Thinking maybe we can go back later and have this be abstracted somehow either like this

```js
componentDidMount() {
    this.fetchData();
}

componentDidUpdate() {
    if (!NetworkUtils.didComponentReconnect(prevProps, this.props)) { 
        return;
    }

    this.fetchData();
}
```
or an HOC like

```js
withNetwork({
    onReconnect: (props) => fetchData();
})(Component);
```

Reply:
Sorry @Justicea83 I put it wrong in the issue description again lol